traffic_portal/test/end_to_end/origins/origins-spec.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var pd = require('./pageData.js');
+var cfunc = require('../common/commonFunctions.js');
+
+describe('Traffic Portal Origins Test Suite', function() {
+    const pageData = new pd();
+    const commonFunctions = new cfunc();
+    const myNewOrigin = {
+        name: "origin-" + commonFunctions.shuffle('abcdefghijklmnopqrstuvwxyz0123456789'),
+        fdqn: "fake.origin.example.com"
+    }
+
+    it('should create a new origin', async () => {
+        browser.setLocation('origins');
+        await pageData.createOriginButton.click();
+        await pageData.name.sendKeys(myNewOrigin.name);
+        commonFunctions.selectDropdownbyNum(pageData.tenant, 1);
```

Comment:
instead of index/number - selecting by option value will be good. So when the element index changes we do not have to worry about anything to fix in script